emcc.py

```diff
@@ -1538,16 +1538,20 @@ def default_setting(name, new_default):
   # errno support by default.
   if settings.MINIMAL_RUNTIME:
     default_setting('SUPPORT_ERRNO', 0)
+    # Require explicit -lfoo.js flags to link with JS libraries.
+    default_setting('AUTO_JS_LIBRARIES', 0)
 
   if settings.STRICT:
     default_setting('STRICT_JS', 1)
-    default_setting('AUTO_JS_LIBRARIES', 0)
```

Comment:
doesn't this removal make strict mode do less than before?

Reply:
Indeed, reverted.   This was part of the change when I was trying to move `default_setting('USE_SDL', 0)` to before the compiler phase where technically it should probably be.